kernel/src/lib.rs

```diff
@@ -134,12 +136,25 @@ fn ap_init() {
 
 fn init_thread() {
     println!("[kernel] Spawn init thread");
+    fs::init_filesystems();
+
+    // TODO: After introducing the mount namespace, use an initial mount namespace to create
+    // the `FsResolver`, and the initial mount namespace should be passed to the first process.
+    init_root_mount();
+    let fs_resolver = FsResolver::new();
+    fs::rootfs::init(
+        boot_info().initramfs.expect("No initramfs found!"),
+        &fs_resolver,
+    )
+    .unwrap();
+    device::init(&fs_resolver).unwrap();
```

Comment:
I don't understand why this FS-related initialization logic should be put in the kernel-wide initialization function.

```rust
    fs::init_filesystems();

    // TODO: After introducing the mount namespace, use an initial mount namespace to create
    // the `FsResolver`, and the initial mount namespace should be passed to the first process.
    init_root_mount();
    let fs_resolver = FsResolver::new();
    fs::rootfs::init(
        boot_info().initramfs.expect("No initramfs found!"),
        &fs_resolver,
    )
    .unwrap();
    device::init(&fs_resolver).unwrap();
```

These lines of code -- FS, mounts, devices --- are all under the FS subsystem. Why not hide them inside the original `fs::lazy_init`?